rs/cross-chain/proposal-cli/src/canister/mod.rs

```diff
@@ -24,6 +25,7 @@ pub enum TargetCanister {
     IcpIndex,
     IcpLedger,
     LedgerSuiteOrchestrator,
+    EvmRpc,
```

Comment:
In the future we can keep extending the enum, but at some point I wonder if generics (i.e. class and object implementing that class) start to make more sense?

Reply:
>i.e. class and object implementing that class

@ninegua That's an interesting idea! Do you mean to be able to use it as a library and deal with canisters that this tool doesn't need to know about?

Reply:
(I will merge it for now to be able to use it for the EVM RPC proposal, but happy to discuss this offline).